x/gamm/keeper/multihop_test.go

```diff
@@ -1,846 +0,0 @@
-package keeper_test
```

Comment:
Note: confirmed that every test case removed in this file has been ported to `x/swaprouter/router_test.go`.

The test fixtures might look different due to clean up. However, every test case is still present